AutomatedTesting/Gem/PythonTests/editor/EditorScripts/AssetBrowser_SearchFiltering.py

```diff
@@ -94,16 +94,21 @@ def verify_files_appeared(model, allowed_asset_extentions, parent_index=QtCore.Q
             Report.info("Asset Browser is already open")
         editor_window = pyside_utils.get_editor_main_window()
         app = QtWidgets.QApplication.instance()
-        
-        # 3) Type the name of an asset in the search bar and make sure only one asset is filtered in Asset browser
+
+        # 3) Type the name of an asset in the search bar and make sure it is filtered to and selectable
         asset_browser = editor_window.findChild(QtWidgets.QDockWidget, "Asset Browser")
         search_bar = asset_browser.findChild(QtWidgets.QLineEdit, "textSearch")
         search_bar.setText("cedar.fbx")
         asset_browser_tree = asset_browser.findChild(QtWidgets.QTreeView, "m_assetBrowserTreeViewWidget")
```

Comment:
why findChild() asset_browser_tree if asset_browser_tree is not used?

Reply:
It's used further down for filtering checks outside of the search context.